Prace_domowe/Praca_domowa4/Marcelina_Kurek/PD4.ipynb

```diff
@@ -0,0 +1,1071 @@
+{
```

Comment:
<p>Kroki są wykonane poprawnie, ale użyty został zły model. SCV to Support Vector Classifier służący (jak sama nazwa wskazuje) do klasyfikacji. Jest to problem regresji, więc powinien był być użyty SVR -- Support Vector Regression. Nie jestem pewien czemu sklearn nie sypał błędami otrzymując na wejściu dane do regresji, ale już same wyniki sugerują że coś jest mocno nie tak; takie wartości RMSE są niezwykle duże. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/242/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>